BEST_PRACTICES.md

```diff
@@ -226,88 +218,74 @@ In this way we are sure only a single instance of the client is created and reus
 
 ## Closing Client Connection
 
-In the JS/TS SDK the connection tears down automatically after each request-response cycle and doesn't need the customer to manually close down the connection.
+In the Commercetools JS/TS SDK the connection tears down automatically after each request-response cycle and doesn't need the customer to manually close down the connection.
 
 ## Customizing the Client
 
-The JS/TS client can be customized in a variety of ways, this can be achieved by passing in middleware options that can take different parameter values.
-
-```ts
-import ClientBuilder from '@commercetools/sdk-client-v2'
-...
-
-// see [this](https://commercetools.github.io/nodejs/sdk/api/#middlewares) on how to configure these options
-const authMiddlewareOptions = {}
-const httpMiddlewareOptions = {}
-
-const client = new ClientBuilder()
-  .withAuthMiddlewareOption(authMiddlewareOptions)
-  .withHttpMiddlewareOption(httpMiddlewareOptions)
-  ...
-  .build()
-```
+The Commercetools JS/TS client can be customized in a variety of ways, this can be achieved by passing in middleware options that can take different parameter values.
+Refer to the [Client Initialization](#client-initialization) section for details on how to create the client.
+To see how to configure the available middleware options and their configurations, see [this documentation](https://commercetools.github.io/nodejs/sdk/api/#middlewares).
```

Comment:
Can we update this link to the link in the official documentation. I believe that is a more updated way of doing things.